tools/destination-tester/README.md

```diff
@@ -153,3 +153,9 @@ This option disables encryption and compression of batch files for debugging pur
 
 #### --input-file
 The tester by default reads all input files from local data folder and executes them in the alphabetical order they appear. You can specify a single input file to be read and executed using this option. Providing just the filename is sufficient.
+
+#### --schema-name
+The tester by default creates `tester` named schema. This option allows the tester to run with different schema name.
```

Comment:
@fivetran-abdulsalam Is the schema named `tester`, literally, or `tester` is used as a prefix or suffix, or the actual name is passed as a tester argument?

Reply:
It is literally `tester`